build.sbt

```diff
@@ -1,7 +1,7 @@
 organization := "com.sergigp.quasar"
 name := "quasar"
 
-version := "0.1.1"
+version := "0.2"
```

Comment:
This is a breaking change, we should increase a version

Reply:
if version < 1.0 == no estable

Reply:
Our love is stable <3